README.md

```diff
@@ -1 +1,109 @@
-# angular-concepts-readme
+# Angular Core Concepts
+
+## Overview
+
+In this lesson, we are going to look at the core concepts that we will use in every Angular application. Some will be familiar to you, however some may seem a bit strange. Don't worry if they take time to sink in!
+
+## Objectives
```

Comment:
Give a little bit more context around the objectives - for example, "Define MVC (Model-View-Controller) and MVVM (Model-View-ViewModel) as they relate to single page web applications". Without a conditional or context, the objectives can be interpreted very broadly and the student can get confused.